deployments/anvilprod/environment.py

```diff
@@ -131,6 +131,73 @@ def mkdict(previous_catalog: dict[str, str],
     mksrc('datarepo-1cb73890', 'ANVIL_cmg_broad_heart_ware_wes_20221215_ANV5_202304242145', 40),
 ]))
 
+anvil2_sources = mkdict({}, 50, mkdelta([
```

Comment:
Question: is omitting the `anvil1` source intentional? For HCA, we usually include the sources from the previous catalog(s) when we add a new one.

Reply:
No, the omission was not intentional. Good catch.